src/controller/CommissioningDelegate.h

```diff
@@ -679,6 +679,16 @@ struct GeneralCommissioningInfo
     ;
 };
 
+struct ICDManagementClusterInfo
```

Comment:
From @bzbarsky-apple I see that there is an issue about documenting, I would just like to add that this struct would benefit from documentation as well, what it's used for, etc.

Reply:
updated, thanks